### PR TITLE
Add copy button

### LINKED ISSUE #15: Add button to copy _targets.R script
## Prework

* [x] I understand and agree to `targetsketch`' [code of conduct](https://github.com/wlandau/targetsketch/blob/main/CODE_OF_CONDUCT.md).
* [x] I understand and agree to `targetsketch`' [contributing guidelines](https://github.com/wlandau/targetsketch/blob/main/CONTRIBUTING.md).
* [x] New features take time and effort to create, and they take even more effort to maintain. So if the purpose of the feature is to resolve a struggle you are encountering personally, please consider first posting a "trouble" or "other" issue so we can discuss your use case and search for existing solutions first.

## Proposal

Please describe the new feature. If applicable, write a minimal example in R code or pseudo-code to show input, usage, and desired output. To help us read any code you include (optional) please try to follow the [tidyverse style guide](https://style.tidyverse.org/). The `style_text()` and `style_file()` functions from the [`styler`](https://github.com/r-lib/styler) package make it easier.

Add button to Control card to copy code from _targets.R card. Copy using [rclipboard](https://github.com/sbihorel/rclipboard) or custom js function.

### PR DESCRIPTION
**Pull request by parmsam**
# Prework

* [x] I understand and agree to this repository's [code of conduct](https://github.com/wlandau/targetsketch/blob/main/CODE_OF_CONDUCT.md).
* [x] I understand and agree to this repository's [contributing guidelines](https://github.com/wlandau/targetsketch/blob/main/CONTRIBUTING.md).
* [x] I have already submitted an issue to the [issue tracker](http://github.com/wlandau/targetsketch/issues) to discuss my idea with the maintainer.

# Summary

Please explain the purpose and scope of your contribution.

Add button to copy _targets.R code to clipboard. Fixed #15.

# Related GitHub issues and pull requests

* Ref: #15

# Checklist

* [x] This pull request is not a [draft](https://github.blog/2019-02-14-introducing-draft-pull-requests).

### CODE REVIEW

File: DESCRIPTION [hunk 1]
```diff
@@ -24,19 +24,20 @@ Authors@R: c(
...
+    shinyalert (>= 1.1),
+    shinyAce (>= 0.4.1),
+    shinybusy (>= 0.2.0),
+    shinycssloaders (>= 0.3),
+    targets (>= 0.2.0),
+    tarchetypes (>= 0.1.0),
+    tidyverse (>= 1.3.0),
+    visNetwork (>= 2.0.9),
+    withr (>= 2.2.0),
+    rclipboard
```
Comment (wlandau) [inline DESCRIPTION:40]: It's a little thing, but would you re-alphabetize the packages and restore the indentation to 2 spaces?

File: R/ui.R [hunk 2]
```diff
@@ -21,7 +22,8 @@ ui <- function() {
               "Update",
               icon = shiny::icon("redo-alt")
             ),
-            shiny::downloadButton("download", "Download")
+            shiny::downloadButton("download", "Download"),
+            uiOutput("clip")
```
Comment (wlandau) [inline R/ui.R:26]: Would you write `shiny::` in front of `uiOutput()`.

Comment (wlandau) [reply R/ui.R:26]: Also, it looks like the copy button is on a new row. Would be great to have the control buttons on the same row.

**Review by wlandau — CHANGES_REQUESTED** [review]
Looks really good. My comments are minor.

**Comment by wlandau** [comment]
Also, feel free to add yourself to the `DESCRIPTION` as a contributor. Example: https://github.com/ropensci/targets/blob/4f44c7729dcbd215977e8c3ebc9c077ac4e23295/DESCRIPTION#L30-L36

File: DESCRIPTION (hunk 1 above)
Comment (parmsam) [reply DESCRIPTION]: Should be resolved now.

File: R/ui.R (hunk 2 above)
Comment (parmsam) [reply R/ui.R]: Sounds good. Using shiny:div() to get buttons on same row.

**Review by wlandau — APPROVED** [review]
Looks great, thanks for the PR.

**Pull request merged**